modules/gatsby/src/components/layout/header.jsx

```diff
@@ -99,11 +98,12 @@ export default class Header extends Component {
               <a key={link.href} href={link.href}>{link.name}</a>
             )) */
     }
-
+    // TODO - let's create the links server side, then pass them to the template as props.
     this.state = {
       links: [
         ...(hasExamples(props) ? [{ label: 'Examples', to: '/examples' }] : []),
         { label: 'Documentation', to: '/docs' },
+        { label: 'Search', to: '/search' },
```

Comment:
Do we want this to be configurable in `ocular-config`? Or we always have search, the question is just if it is search-as-a-service or pre-generated?

Reply:
Yes I absolutely want this to be in ocular-config. This and all the other links. I don't think we can have a systematic link to vis.gl medium blog on the header :) plus people can add links to story book, or other custom parts of website